docker/gcp_pudl_etl.sh

```diff
@@ -70,7 +70,7 @@ function run_pudl_etl() {
 function save_outputs_to_gcs() {
     echo "Copying outputs to GCP bucket $PUDL_GCS_OUTPUT" && \
     gsutil -m cp -r "$PUDL_OUTPUT" "$PUDL_GCS_OUTPUT" && \
-    rm "$PUDL_OUTPUT/success"
+    rm -f "$PUDL_OUTPUT/success"
```

Comment:
If there's no `success` file this exits with a non-zero code, which is not what we want.

Reply:
omg this has been bugging me for literally days